modules/ingest-geoip/src/test/java/org/elasticsearch/ingest/geoip/DatabaseNodeServiceTests.java

```diff
@@ -109,6 +114,8 @@ public class DatabaseNodeServiceTests extends ESTestCase {
     private IngestService ingestService;
     private ClusterService clusterService;
 
+    private final Collection<Releasable> toRelease = new CopyOnWriteArrayList<>();
```

Comment:
No the most elegant solution here but the pattern IMO makes sense and we could think about using it elsewhere where managing this kind of reference isn't so trivial